monthly/src/sandbox.ts

```diff
@@ -1,29 +1,32 @@
 // import {prisma} from "./server/context";
-import {queue} from "./lib/queue";
-// import {basename} from "path";
+import {seedDummyFiles} from "./files/seed";
```

Comment:
Create a `sandbox.exampel.ts` and ignore this one and explain about in the readme